doc/cli/operator-sdk_alpha.md

```diff
@@ -15,5 +15,6 @@ Run an alpha subcommand
 ### SEE ALSO
 
 * [operator-sdk](operator-sdk.md)	 - An SDK for building operators with ease
+* [operator-sdk alpha bundle](operator-sdk_alpha_bundle.md)	 - Operator bundle commands
```

Comment:
`Operator bundle commands`?  Could we improve this description in order to make more clear its purpose?